### PR TITLE
build 0.44b0 :snowflake:

### PR DESCRIPTION
**Pull request by boldorider4**
opentelemetry-semantic-conventions 0.44b0 :snowflake:

**Destination channel:** Snowflake

### Links

- [PKG-4252]
- [Upstream repository](https://github.com/open-telemetry/opentelemetry-python/tree/main/opentelemetry-semantic-conventions)

[PKG-4252]: https://anaconda.atlassian.net/browse/PKG-4252?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,27 +1,28 @@
...
   skip: true  # [py<36]
-  # noarch: python
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   host:
     - pip
     - python
     - setuptools
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,27 +1,28 @@
...
     - wheel
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
```
not required, because hatchling is used

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,27 +1,28 @@
...
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/opentelemetry-semantic-conventions-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ alt_name }}-{{ version }}.tar.gz
   sha256: {{ sha256 }}
 
 build:
   number: 0
   skip: true  # [py<36]
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
  skip: true  # [py<38]
```
https://github.com/open-telemetry/opentelemetry-python/blob/v1.24.0/opentelemetry-semantic-conventions/pyproject.toml#L11

**Review by cbouss — APPROVED** [review]

File: recipe/meta.yaml [hunk 4]
```diff
@@ -35,18 +34,19 @@ test:
...
+  description: |
     OpenTelemetry, also known as OTel for short, is a vendor-neutral open-source
     Observability framework for instrumenting, generating, collecting, and exporting
     telemetry data such as traces, metrics, logs. As an industry-standard
     it is natively supported by a number of vendors.
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE
+  doc_url: https://opentelemetry-python.readthedocs.io
+  doc_source_url: https://github.com/open-telemetry/opentelemetry-python
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
```
`doc_url` is enough

**Review by skupr-anaconda — APPROVED** [review]
LGTM but a redundant `doc_source_url`

**Comment by ViridianMelody** [comment]
The latest version 0.45.b0 is not compatible with the rest of the `opentelemetry 1.23.0` suite and doesn't match the version Snowflake requested. Fortunately, there's no difference in the requirements, so I'm switching this build to version 0.44.b0 to try and push these through before sprint end.

**Review by JeanChristopheMorinPerso — APPROVED** [review]

**Review by cbouss — APPROVED** [review]

**Pull request merged**